src/MasterSite.php

```diff
@@ -252,35 +251,6 @@ function () {
         // Make post tags ordered.
         add_filter('register_taxonomy_args', [$this, 'set_post_tags_as_ordered'], 10, 2);
 
-        add_action(
-            'customize_register',
-            function ($wp_customize): void {
-                // Add new site identity styles toggle.
-                $wp_customize->add_setting('new_identity_styles', ['default' => false]);
-                $wp_customize->add_control(new WP_Customize_Control(
-                    $wp_customize,
-                    'new_identity_styles',
-                    array(
-                        'label' => __('Enable new Greenpeace visual identity', 'planet4-master-theme'),
-                        'description' => __(
-                            'This opt-out setting will be phased out at the end of September',
-                            'planet4-master-theme'
-                        ),
-                        'settings' => 'new_identity_styles',
-                        'type' => 'checkbox',
-                        'section' => 'title_tagline',
-                    )
-                ));
-
-                if (!defined('WP_APP_ENV') || ('production' !== WP_APP_ENV && 'staging' !== WP_APP_ENV)) {
```

Comment:
I think we should keep this part to disable the CSS customizer, it's not related to the new identity